### PR TITLE
Adding my first homework in JS week 1

### PR DESCRIPTION
**Pull request by Bogdy87**
All the code runs well, but it can be written in a shorter way with functions. However, I choose to keep it like this, taking step by step.

### CODE REVIEW

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+              --FreeCodeCamp--
+      https://www.freecodecamp.org/BogdyAlex87*/
+
+//*************************************************************/
+
+/*Step 3: Smart-ease
+          Age-ify (A future age calculator)*/
+//******************************************************************
+
+let yearOfBirth = 1987;
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js]: Use const by default and switch to let only if necessary

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+let yearFuture = 2059;
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js]: same here

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+let age = yearFuture - yearOfBirth;
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js]: const

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+console.log("You will be " + age + " years old in " + yearFuture);
+
+/*Goodboy-Oldboy (A dog age calculator)*/
+
+const dogYearOfBirth = 2020;
+let dogYearFuture = 2024;
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js]: const

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+let humanAge = dogYearFuture - dogYearOfBirth;
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js]: const

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+if (humanAge === 1) {
+  dogAge = 15;
+} else if (humanAge === 2) {
+  dogAge = 24;
+} else {
+  dogAge = 24 + (humanAge - 2) * 5;
+}
+
+let shouldShowResultInDogYears = true;
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js]: const

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+let dogYear = shouldShowResultInDogYears ? dogAge : humanAge;
+console.log(
+  "Your dog will be ",
+  dogYear,
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js:38]: const

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+  dogAge = 15;
+} else if (humanAge === 2) {
+  dogAge = 24;
+} else {
+  dogAge = 24 + (humanAge - 2) * 5;
+}
+
+let shouldShowResultInDogYears = true;
+
+let dogYear = shouldShowResultInDogYears ? dogAge : humanAge;
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js]: const

**Comment by Bogdy87** [comment]
Thanks for feedback. I've changed the let with const and only one I couldn't change.

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+  "RANDOM -The startup: " +
+    randomStartupName +
+    " contains " +
+    randomStartupName.length +
+    " characters."
+);
+                  //  Generate a fixed startup name by given a index number for each array
+const startupName = firstWords[1] + " " + secondWords[6];
+
+console.log(
```
Comment (aina21) [inline javascript/javascript1/week1/Homework/smartEase.js:144]: You can also use a template literal for better readability, like this:
```js
`FIXED-The startup: ${startupName} contains ${startupName.length} characters.`
```

**Review by aina21 — APPROVED** [review]

File: javascript/javascript1/week1/Homework/smartEase.js (hunk 9 above)
Comment (Bogdy87) [reply javascript/javascript1/week1/Homework/smartEase.js:144]: Updated and ready to merge!

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+
+let dogAge;
+
+if (humanAge === 1) {
+  dogAge = 15;
+} else if (humanAge === 2) {
+  dogAge = 24;
+} else {
+  dogAge = 24 + (humanAge - 2) * 5;
+}
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/Homework/smartEase.js:31]: 1 dog year is === 7 human years according to the assignment description. 
> "Your dog will be 10 human years old in 2027" or "Your dog will be 70 dog years old in 2027"

You might need to update your code here.

File: javascript/javascript1/week1/Homework/smartEase.js [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+  dogYear,
+  shouldShowResultInDogYears
+    ? " dog years old in " + dogYearFuture
+    : "human year old in " + dogYearFuture
+);
+
+/*Housey pricey (A house price estimator)*/
+
+// Peter's house details
+const peterWidth = 8;
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/Homework/smartEase.js:47]: Those variable names can be confusing unless someone is familiar with the assignment. `peterHouseWidth` might be more precise.

**Review by jason-vasilev — COMMENTED** [review]
You are right about functions. It will be much easier and cleaner to complete the House Price task.
I've left you 2 small comments. If you want to fix them, first checkout `main` get latest changes and then checkout to a new branch, else you risk of getting quickly familiar with merge conflicts.

File: javascript/javascript1/week1/Homework/smartEase.js (hunk 10 above)
Comment (Bogdy87) [reply javascript/javascript1/week1/Homework/smartEase.js:31]: Updated.

File: javascript/javascript1/week1/Homework/smartEase.js (hunk 11 above)
Comment (Bogdy87) [reply javascript/javascript1/week1/Homework/smartEase.js:47]: Changed.

**Pull request merged**